src/Network/Network/Models/PSLoadBalancerBackendAddress.cs

```diff
@@ -0,0 +1,40 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+using Microsoft.Azure.Management.Network.Models;
+
+using Microsoft.WindowsAzure.Commands.Common.Attributes;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.Commands.Network.Models
+{
+    public partial class PSLoadBalancerBackendAddress : PSChildResource
+    {
+        [JsonProperty(Order = 1)]
+        [Ps1Xml(Target = ViewControl.Table)]
+        public PSNetworkInterfaceIPConfiguration BackendIpConfigurations { get; set; }
+
+        [JsonProperty(Order = 1)]
+        public PSVirtualNetwork VirtualNetwork { get; set; }
+
+        [JsonProperty(Order = 1)]
+        public string IpAddress { get; set; }
+
```

Comment:
do we need to check the combination, as we dicussed?

Reply:
No I don't think so, right now we don't populate NIC config if provided, so PsLoadBalancerBackendAddress will always have a IpAddress and VNET ID, other wise an error is returned by New-AzLoadBalancerBackendAddressConfig which creates this object.

Reply:
the fact that we cannot modify does not mean that we can see it as read only

Reply:
it is read only if available now.